src/components/About/Blurbs.js

```diff
@@ -0,0 +1,82 @@
+import React, { Component } from 'react';
+
+class Blurbs extends Component {
+  render() {
+    return (
+      <div>
+
+        <h1 id = "title" >
+          our story
+        </h1>
+
+        <div style = {{padding: "20px 0px 20px 0px"}}>
```

Comment:
It is good practice to put styles `<div style = {{padding: "20px 0px 20px 0px"}}>` in CSS files (in this case About.css) to ease future extensions/amendments.